tests/dash/app_dropdown.py

```diff
@@ -13,7 +13,8 @@
 ID_PREFIX = "app_dropdown"
 IDS = {
     "dropdown": ID_PREFIX,
-    "dropdown-by-cell": '{}-row-by-cell'.format(ID_PREFIX)
+    "dropdown-by-cell": '{}-row-by-cell'.format(ID_PREFIX),
+    "dropdown-by-cell-deprecated": '{}-deprecated-row-by-cell'.format(ID_PREFIX)
```

Comment:
Not certain what happen but I think the recent change to renderer (keys/ids) is responsible for the regression -- both table had the same id -- apparently causing the second one to disappear from the DOM entirely.

Reply:
https://github.com/plotly/dash-renderer/pull/101

Reply:
release-v1 did not have this problem because the latest master changes haven't been merged in yet -- process yet to be put in place :)

Reply:
Oh interesting - that's probably going to bite us and our users a lot of places. Should we make it an exception on the Python side if you ever provide multiple components with matching id?

Reply:
> make it an exception on the Python side if you ever provide multiple components with matching id?

There is a duplicate ids check: https://github.com/plotly/dash/pull/320

Reply:
> There is a duplicate ids check: plotly/dash#320

why wasn't that triggered here?

Reply:
Layout is dynamic, it doesn't work when that is the case.